Cargo.toml

```diff
@@ -81,6 +80,18 @@ serde = [
     "dep:serde",
     "ipnet/serde",
 ]
+native-tls = [
+    "oneio/lib",
+    "oneio/native-tls"
```

Comment:
This is duplicative, but I thought might be a little more expressive:

https://github.com/bgpkit/oneio/blob/f2f268eea366211cbc07c053f1238d00077c908e/Cargo.toml#L50C8-L50C56